### PR TITLE
<Checkbox/> - errorMessage depricate, tooltipContent and tooltipProps was added

### PR DESCRIPTION
**Pull request by roibWix**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by roibWix** [comment]
https://jira.wixpress.com/browse/DSM-1323

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6438

File: src/Checkbox/Checkbox.js [hunk 1]
```diff
@@ -95,13 +105,14 @@ class Checkbox extends React.PureComponent {
           >
             <Tooltip
               dataHook={dataHooks.boxTooltip}
-              disabled={disabled || !hasError || !errorMessage}
+              disabled={isDisabled}
+              content={tooltipContent || errorMessage || ' '}
               placement="top"
```
Comment (EgoziE) [inline src/Checkbox/Checkbox.js:110]: can remove placement="top" this is already the default

**Review by EgoziE — APPROVED** [review]
Looks good, just a small comment

**Pull request merged**